### PR TITLE
add HtoZA signal files

### PR DESCRIPTION
**Pull request by alesaggio**

### CODE REVIEW

File: analyses/HtoZA.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "HtoZAAnalysis",
+    "configuration": "cp3_llbb/ZAAnalysis/test/HtoZAConfiguration.py",
+    "splitting": "relative:1",
+    "samples": {
+        "mc": [
+            "HToZA", "DY_NLO", "DY_NLO_NJetsBins", "TT_powheg_inclusive", "ZZ"
```
Comment (OlivierBondu) [inline analyses/HtoZA.json]: would you mind adding at least ZH and single top ?

**Review by OlivierBondu — CHANGES_REQUESTED** [review]
thanks ! only minor addition missing :)

File: analyses/HtoZA.json (hunk 1 above)
Comment (alesaggio) [reply analyses/HtoZA.json]: Which one of the single tops? There are three possible choices: ST_tW_5f_powheg, ST_s_4f_aMC@NLO, ST_t_4f_powheg.
And I can't seem to find the ZH in there...

Comment (OlivierBondu) [reply analyses/HtoZA.json]: all three production modes :)

for ZH, it should be hidden in https://github.com/cp3-llbb/Datasets/blob/RunIISummer16MiniAODv2/datasets/mc_SMHiggs.json, but you're right, I'll do a finer splitting of this file, right now this is a bit impractical, so forget about it, I'll do a PR later

Comment (alesaggio) [reply analyses/HtoZA.json]: ok :)

**Review by OlivierBondu — APPROVED** [review]

**Pull request merged**